sympy/physics/control/lti.py

```diff
@@ -5,13 +5,13 @@
 )
 from sympy.polys.polytools import degree
 
-__all__ = ['TransferFunction']
+__all__ = ['TransferFunctionMatrix', 'TransferFunction']
 
 _matrixTypes = (
     Matrix, ImmutableMatrix, MutableMatrix, MutableDenseMatrix)
 
 
-class TransferFunction(Basic):
+class TransferFunctionMatrix(Basic):
```

Comment:
I'll remove this class when all functionality  of `TransferFunction` is working properly.